src/NzbDrone.Host/NzbDrone.Host.csproj

```diff
@@ -56,9 +56,6 @@
   <PropertyGroup>
     <RunPostBuildEvent>OnOutputUpdated</RunPostBuildEvent>
   </PropertyGroup>
-  <PropertyGroup>
-    <ApplicationIcon>Radarr.ico</ApplicationIcon>

```

Comment:
Why was this removed?

Reply:
Thats was removed and re added more inline how the other 2 forks use .\NzbDrone.Host